docs/Commands.md

```diff
@@ -75,11 +81,20 @@ Indexing books that have a "categories" field in which semicolons separate the v
 FT.CREATE books-idx ON HASH PREFIX 1 book:details FILTER SCHEMA title TEXT categories TAG SEPARATOR ";"
 ```
 
+Indexing a JSON document using JSON Path:
+
+```sql
+FT.CREATE idx ON JSON SCHEMA $.title AS title TEXT $.categories AS categories TAG SORTABLE
+```
+
 #### Parameters
 
 * **index**: the index name to create. If it exists the old spec will be overwritten
 
-* **ON {structure}** currently supports only HASH (default)
+* **ON {structure}** currently supports HASH (default) and JSON.
```

Comment:
{structure} or {data type}?

Reply:
@oshadmi Good point I think we can replace structure by datatype.
@banker  Yes, HASH is still a default. ON not required